packages/block-library/src/post-featured-image/dimension-controls.js

```diff
@@ -143,6 +146,30 @@ const DimensionControls = ( {
 					</ToggleGroupControl>
 				</ToolsPanelItem>
 			) }
+			{ !! imageSizeOptions.length && (
+				<ToolsPanelItem
+					hasValue={ () => !! sizeSlug && sizeSlug !== DEFAULT_SIZE }
```

Comment:
Not a strong opinion, but do we want `hasValue` to be false when the default size is selected? By default and whenever the control is Reset through the ToolsPanel menu, the attribute is `undefined` -- so I believe if `sizeSlug === DEFAULT_SIZE` that is always because of an explicit user selection, and it may be slightly unexpected for the control to be hidden when the block is reselected.

Reply:
Good catch. Thanks, @stacimc.